src/commands/referral/services.ts

```diff
@@ -729,12 +728,16 @@ export const services = [
   'zwift',
 ];
 
+const CUSTOM_SERVICES: string[] = ['everyday rewards (woolworth & big W)'];
+
+export const services: string[] = [...new Set([...OZBARGAIN_SERVICES, ...CUSTOM_SERVICES])];
```

Comment:
deduplicate the service names in the OZBARGAIN scraped list vs our own manual list.